pages/public_cloud/containers_orchestration/managed_kubernetes/expose_your_services_using_a_load_balancer/guide.en-gb.md

```diff
@@ -0,0 +1,399 @@
+
+
+## Prerequisites
+
+#### Kubernetes version
+- To be able to deploy Public Cloud Load Balancers, your Managed Kubernetes Service must run or have been upgraded to the following patch versions:
+
+| Kubernetes versions |
+|-------------|
+| 1.24.13-3>= |   
+| 1.25.9-3>=  |   
+| 1.26.4-3>=  |   
+| 1.27>=      |  
+
+#### Network topologie to expose your Load Balancers publicly   //à reformuler
+If you plan to expose your Load Balancer publicly, in order to attach a FloatingIP to your Load Balancer, an OVHcloud Gateway (an Openstack router) is mandatory on the subnet hosting your Load Balancer.
+For such use case we do recommend to create your MKS clusters on a network and subnet using our OVHcloud Gateway.
+For existing clusters, if:
+- **The Subnet's GatewayIP is already an OVHcloud Gateway**, nothing needs to be done. The current Openstack Router will be used.
+- **The subnet doest not have an IP reserved for a Gateway** --> You will have to provide/create a compatible subnet. You can edit the existing subnet (modop CLI Openstack/Horizon) or use another dedicated subnet (doc LoadBalancerSubnetID)//TODO
+- **The GatewayIP is already assigned to a non-OVHcloud Gateway (Openstack Router)** --> You will have to provide/create a compatible subnet. To do so you select another dedicated subnet (doc LoadBalancerSubnetID)//TODO
+
+
+
+## Limitations
+
+- Layer 7 Policy & Rules and TLS Termination (`TERMINATED_HTTPS` listener) are not available yet. For such use cases you can rely on [Octavia Ingress Controller](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/octavia-ingress-controller/using-octavia-ingress-controller.md)
+- UDP proxy protocol is not supported
+
+
+## Billing
+
+- When exposing your load balancer publicly (pub-to-pub or pub-to-private):
+  - if it does not already exist, an OVHcloud Gateway is automatically created and charged for all Load Balancers spawned in the subnet <https://www.ovhcloud.com/en-gb/public-cloud/prices/#10394>.
+  - a Public Floating IP will be used: https://www.ovhcloud.com/en-gb/public-cloud/prices/#10346
+- Each Public CLoud Load Balancer is billed according to his flavor: <https://www.ovhcloud.com/en-gb/public-cloud/prices/#10420>
+
+    > [!primary]
+    >
+    > Note: Each publicly exposed Load Balancer has his own Public Floating IP. Outgoing traffic doesn't consume OVHcloud Gateway bandwidth.
+    >
+
+    > [!warning]
+    >
+    > During the CCM Beta, since the Public Cloud Load Balancer is GA, the Public Cloud Load Balancer usage as well as the other network components (Gateway & Floating IPs) will be billed
```

Comment:
```suggestion
    > During the Public Cloud Load Balancer with Manage Kubernetes Service Beta, since the Public Cloud Load Balancer is GA, the Load Balancer usages as well as the other network components (Gateway & Floating IPs) will be charged
```